_layouts/website/_footer.html

```diff
@@ -1,63 +1,83 @@
 <div class="primary-nio-footer">
     <footer>
         <div class="links">
-            <div class="contact-info">
-                <div class="logos">
-                    <a href="https://app.n.io">
-                    <svg id="footer-logo" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 1088 360" class="logo--nio">
-                      <defs>
-                        <style>.cls-1,.cls-3{fill:#e6e9ee;}.cls-2{fill:#fff;}.cls-3{font-size:23px;font-family:ArialMT, Arial;}
-                        </style>
-                      </defs>
-                      <title>niolabs logo light</title>
-                      <path class="cls-1" d="M513.3,89.3V299H477V89.3Z"/>
-                      <path class="cls-1" d="M662.7,151.1h35.1V299H663.3l-1.8-21.6c-8.4,17.4-31.5,25.8-48,26.1-43.8.3-76.2-26.7-76.2-78.6,0-51,33.9-77.7,77.1-77.4,19.8,0,38.7,9.3,47.1,24Zm-88.8,73.8c0,28.2,19.5,45,43.8,45,57.6,0,57.6-89.7,0-89.7C593.4,180.2,573.9,196.7,573.9,224.9Z"/>
-                      <path class="cls-1" d="M762.3,89.3v81.9c8.7-15.3,33-23.7,48.6-23.7,43.2,0,75.3,26.4,75.3,77.4,0,48.6-32.7,77.4-76.2,77.4-18,0-36.3-6-47.7-23.7L759.9,299H725.7V89.3Zm2.4,135.6c0,26.7,19.8,43.5,42.9,43.5,23.4,0,42-17.7,42-43.5,0-26.7-18.6-43.2-42-43.2C784.5,181.7,764.7,199.1,764.7,224.9Z"/>
-                      <path class="cls-1" d="M1009.8,190.4c-10.5-9.9-22.5-13.2-36.6-13.2-17.4,0-27,5.4-27,14.7,0,9.6,8.7,15,27.6,16.2,27.9,1.8,63.3,8.1,63.3,47.4,0,26.1-21.3,48.6-63.6,48.6-23.4,0-46.8-3.9-68.4-26.4l18-26.1c10.5,11.7,34.5,20.4,51,20.7,13.8.3,26.7-6.9,26.7-17.7,0-10.2-8.4-14.4-29.4-15.6-27.9-2.1-61.2-12.3-61.2-45.9,0-34.2,35.4-46.2,62.4-46.2,23.1,0,40.5,4.5,57.6,19.5Z"/>
-                      <path class="cls-2" d="M161.69,299.43V221.72c0-22.59-12.35-39.76-35.84-39.76-22.59,0-38,19-38,41.56v75.9H51.45V150.64H84.28l2.41,20.18c15.06-14.76,30.12-22.29,48.79-22.29,34.94,0,63,26.21,63,72.89v78Z"/>
-                      <path class="cls-2" d="M269,109.07c0,28.61-43.37,28.61-43.37,0S269,80.45,269,109.07ZM229,150.34V299.43h36.74V150.34Z"/>
-                      <path class="cls-2" d="M449.74,225.33c0,42.78-29.22,77.41-77.41,77.41s-77.11-34.63-77.11-77.41c0-42.47,29.52-77.41,76.81-77.41S449.74,182.87,449.74,225.33Zm-117.77,0c0,22.6,13.55,43.68,40.36,43.68s40.36-21.08,40.36-43.68c0-22.29-15.66-44-40.36-44C345.83,181.36,332,203,332,225.33Z"/>
-                      <text class="cls-3" transform="translate(1042.83 144.46)">™
-                      </text>
-                    </svg>
+            <div class="col-md-12 col-lg-4 contact-info">
+                <div class="logos flex-wrap">
+                    <a href="https://niolabs.com" class="footer__logo logo--niolabs">
```

Comment:
@jennyknuth I think we want it to go to app.n.io from docs

Reply:
@jennyknuth shoot, sorry, the footer should be niolabs, nevermind, havent had my coffee yet